api_orders.md

```diff
@@ -56,6 +59,21 @@ This documentation describes the snabble API endpoints related to the
 | `price.taxPre`         | `map[string]int`  | Mapping of tax rates sums all pre tax prices of products with this rate up                    |
 | `session`              | `string`          | The session ID                                                                                |
 | `fiscalReference`      | `FiscalReference` | Optional [Fiscal Reference](#fiscal-reference)                                                |
+| `cashRegisterID`       | `string`          | The ID of the virtual cash register used.                                                     |
+| `sequenceNumber`       | `int`             | Sequence number of the order                                                                  |
+| `externalCheckoutID`   | `string`          | Optional. The ID of the order in an external system in case the order was processed via an external system. (eg.VPoS) |
```

Comment:
```suggestion
| `externalCheckoutID`   | `string`          | Optional. The ID of the order in an external system in case the order was processed via an external system. (eg. vPoS) |
```